spec/requests/vendor_api/reject_application_spec.rb

```diff
@@ -3,6 +3,19 @@
 RSpec.describe 'Vendor API - POST /applications/:application_id/reject', type: :request do
   include VendorApiSpecHelpers
 
+  let(:valid_metadata) {
+    {
+      attribution: {
+        full_name: 'Jane Smith',
+        email: 'jane@example.com',
+        user_id: '12345',
+      },
+      timestamp: Time.now.iso8601,
+    }
+  }
```

Comment:
Shall we refactor this into the helper? The repetition is signalling that `valid_metadata` has a different meaning across the different requests